core/src/main/java/com/nisovin/magicspells/util/VariableMod.java

```diff
@@ -12,19 +12,19 @@
 public class VariableMod {
 	
 	public enum VariableOwner {
-		
 		CASTER,
 		TARGET
-		
 	}
 	
 	public enum Operation {
 		
 		SET((a, b) -> b),
-		ADD((a, b) -> a + b),
+		ADD(Double::sum),
 		MULTIPLY((a, b) -> a * b),
 		DIVIDE((a, b) -> a / b),
-		POWER((a, b) -> FastMath.pow(a, b));
+		MODULO((a, b) -> a % b),
+		POWER(FastMath::pow),
+		RANDOM((a, b) -> Double.valueOf(Util.getRandomInt(b.intValue())));
```

Comment:
How was the decision reached to lock it to integer values?

Reply:
With lack of experience...